<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordChangeForm</class>
 <widget class="QWidget" name="PasswordChangeForm">
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0"><widget class="QLabel" name="label_pin"><property name="text"><string>New PIN</string></property></widget></item>
   <item row="0" column="1"><widget class="QLineEdit" name="newPinEdit"><property name="echoMode"><enum>QLineEdit::Password</enum></property></widget></item>
   <item row="1" column="0"><widget class="QLabel" name="label_pass"><property name="text"><string>New Second Password</string></property></widget></item>
   <item row="1" column="1"><widget class="QLineEdit" name="newSecondPassEdit"><property name="echoMode"><enum>QLineEdit::Password</enum></property></widget></item>
   <item row="2" column="1"><widget class="QPushButton" name="applyBtn"><property name="text"><string>Apply</string></property></widget></item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>